actionpack/lib/action_controller/caching.rb

```diff
@@ -9,17 +9,16 @@ module ActionController
   # of calculations, renderings, and database calls around for subsequent
   # requests.
   #
-  # You can read more about each approach by clicking the modules below.
-  #
   # Note: To turn off all caching provided by Action Controller, set
+  #
   #     config.action_controller.perform_caching = false
   #
   # ## Caching stores
   #
-  # All the caching stores from ActiveSupport::Cache are available to be used as
+  # All the caching stores from ActiveSupport::Cache::Store are available to be used as
```

Comment:
This is not right, because all the cache stores are under `ActiveSupport::Cache` namespace.

Reply:
Ah, I see they are all listed in the namespace there ... removed this one.